tests/components/deconz/test_cover.py

```diff
@@ -75,15 +74,15 @@ async def test_cover(hass):
     data = deepcopy(DECONZ_WEB_REQUEST)
     data["lights"] = deepcopy(COVERS)
     gateway = await setup_deconz_integration(hass, get_state_response=data)
-    assert "cover.level_controllable_cover" in gateway.deconz_ids
-    assert "cover.window_covering_device" in gateway.deconz_ids
-    assert "cover.unsupported_cover" not in gateway.deconz_ids
-    assert "cover.deconz_old_brightness_cover" in gateway.deconz_ids
-    assert "cover.window_covering_controller" in gateway.deconz_ids
+
     assert len(hass.states.async_all()) == 5
+    assert hass.states.get("cover.level_controllable_cover").state == "open"
+    assert hass.states.get("cover.window_covering_device").state == "closed"
+    assert hass.states.get("cover.unsupported_cover") is None
+    assert hass.states.get("cover.deconz_old_brightness_cover").state == "open"
+    assert hass.states.get("cover.window_covering_controller").state == "closed"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)